assets/javascripts/discourse/controllers/user-activity-reactions.js

```diff
@@ -61,18 +63,18 @@ export default Controller.extend({
     CustomReaction.findReactions(this.reactionsUrl, this.username, opts)
       .then((newReactionUsers) => {
         reactionUsers.addObjects(newReactionUsers);
-        this._updateBeforeIds(newReactionUsers);
+        this.#updateBeforeIds(newReactionUsers);
         if (newReactionUsers.length === 0) {
-          this.set("canLoadMore", false);
+          this.canLoadMore = false;
         }
       })
       .finally(() => {
-        this.set("loading", false);
+        this.loading = false;
       });
-  },
+  }
 
   @observes("canLoadMore")
   _showFooter() {
-    this.set("application.showFooter", !this.canLoadMore);
```

Comment:
`ApplicationController` hasn't been converted to use `@tracked` yet, so I think we've gotta stick with `.set()` here for now